bulgarian/free_runner/tree/adrenaline/adrenaline.md

```diff
@@ -0,0 +1 @@
+Имаш инжекция с адреналин. Биеш си я в сърцето. Силите ти се възвръщат, забравяш за болката и продължаваш да бягаш. Зад теб се появява кола.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.